coarse/services/typo_grammar_agent.py

```
@@ -0,0 +1,15 @@
+from langchain.output_parsers import OutputFixingParser
+from langchain_core.output_parsers import JsonOutputParser
+from fastapi import UploadFile
+import os
+from coarse.constants.typo_grammar import TYPO_GRAMMAR_PROMPT
+from coarse.constants.coarse import llm
+from coarse.models.errors import Errors
+
+async def check_typo_grammar(paper_obj):
+    chain = TYPO_GRAMMAR_PROMPT | llm | OutputFixingParser.from_llm(parser=JsonOutputParser(pydantic_object=Errors), llm=llm)
```

Pylint (3.3.7)

[convention] Line too long (125/100)

(C0301)